lavalink/WebSocket.py

```diff
@@ -1,16 +1,19 @@
 import asyncio
-import websockets
 import json
 import logging
-from .Events import TrackStuckEvent, TrackExceptionEvent, TrackEndEvent
+from datetime import datetime
+
+import websockets
 
+from .Events import TrackStuckEvent, TrackExceptionEvent, TrackEndEvent
 
 log = logging.getLogger(__name__)
 
 
 class WebSocket:
     def __init__(self, lavalink, host, password, ws_port, ws_retry, shard_count):
         self._lavalink = lavalink
+        self.last_response = 0

```

Comment:
The value looks to be updated but never queried. Does this have a purpose?

Reply:
It's a debug leftover. Sorry.